bids/events/base.py

```diff
@@ -250,24 +250,25 @@ def __init__(self, base_dir, default_duration=0, default_amplitude=1,
         self.extra_columns = extra_columns
         self.sampling_rate = sampling_rate
 
-    def read(self, files=None, reset=True, **kwargs):
+    def read(self, file_directory=None, reset=True, **kwargs):
         ''' Read in and process event files.
         Args:
-            files (None, list): Optional list of event files to read from. If
-                None (default), will use all event files found within the
-                current BIDS project. Note that if `files` is passed, existing
-                event files within the BIDS project are completely ignored.
+            file_directory (None, str): Optional path to folder containing
```

Comment:
Good point. Okay, I agree with your comment in #49: this probably requires that we extract entities from the incoming file. Let's go with your current approach for the moment, and I'll think about the best way to handle this more generally (likely by adding an `add_file` method to the grabbit `Layout` class).